src/controllers/match-experience.controller.ts

```diff
@@ -19,10 +19,28 @@ export const matchExperienceController = {
 
     getAll: async (req: Request, res: Response) => {
```

Comment:
```suggestion
[    getAll: async (req: Request, res: Response) => {](req: Request<unknown, unknown, unknown, GetAllExperiencesParams>)
```

this way it would be type safe and you want need to make cast to string each time (`as string`) when using `req.query`